### DISCUSSION TITLE
RapidVideOCR cannot recognize some Chinese characters

### DISCUSSION THREAD

**Discussion by chuducnam** [SWHL/RapidVideOCR#61] [Q&A] [answered]
When i try to create subtitles from videos of the comedy group "Chen Xiang liu dian ban", some chinese characters are not recognized by RapidVideOCR. Specifically, the following characters are: 恭喜, 嗯, 喂, 犟...
I only tested with 2 videos, as the number of video increases, I think the number of characters that RapidVideOCR fails to recognize might also increase.
What could be the issue here? Please help me understand.
非常感谢您的支持与帮助!
![img1](https://github.com/user-attachments/assets/aa3e9ffc-fda4-4a7f-beff-3d7679f9d94d)

**Comment by SWHL** [comment]
Please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).

**Reply by chuducnam** [reply]
RapidVideOCR fails to recognize the Chinese characters 恭喜, 嗯, and 喂 in the provided video.

00:01:49,520 --> 00:01:50,879 恭喜
00:02:32,840 --> 00:02:33,639 嗯
00:03:54,680 --> 00:03:55,159 喂
00:05:13,400 --> 00:05:14,039 嗯

Video URL: https://www.douyin.com/video/7397378730076818697

RapidVideOCR version 2.2.13
OCR Library RapidOCR-ONNXRuntime version 1.3.24

result.srt and RGBImages attached below

[RGBImages.zip](https://github.com/user-attachments/files/16439208/RGBImages.zip)
[result.zip](https://github.com/user-attachments/files/16439220/result.zip)

![test 1](https://github.com/user-attachments/assets/50ee7bd2-3f4c-4801-bbb4-32105c0b4d5d)

Thank you very much!

**Reply by SWHL** [reply, marked as answer]
I have deeply analyzed the reasons why single words cannot be recognized. This problem is a problem with the OCR recognition model. This problem is quite common. OCR cannot recognize 100% correctly, and manual correction is basically required in the later stage.

I have raised an [issue](https://github.com/RapidAI/RapidOCR/issues/210) under the RapidOCR project, hoping to alleviate it through strategies.

**Reply by chuducnam** [reply]
Hi SWHL,
I'm encountering a similar issue, but with images containing more than one character.

When running the RapidOCR model, despite images having identical characters (but different backgrounds), the returned results differ. Additionally, some characters are recognized incorrectly or not recognized at all. 

Of course, expecting a model to achieve 100% accuracy is unrealistic, but could you please explain this issue to me again? 


Thank you very much for your time!
![ocr](https://github.com/user-attachments/assets/cd426563-de96-469b-a87b-67bfe2d58bd7)
[img.zip](https://github.com/user-attachments/files/16517825/img.zip)
